skipper.yaml

```diff
@@ -8,5 +8,6 @@ volumes:
   - $HOME/.ssh/known_hosts:/root/.ssh/known_hosts:ro
   - ${XDG_RUNTIME_DIR}/containers/auth.json:/run/user/0/containers/auth.json:ro
   - /var/ai-logs:/var/ai-logs # using this when downloading triage logs
+  - $HOME/triage-tools-tickets:$HOME/triage-tools-tickets
```

Comment:
Please consider not messing with user's home for this purpose. Why don't we create a directory (in .gitignore) within the repo's path?